openhtf/core/dut_id.py

```diff
@@ -13,3 +13,12 @@ class DutIdentifier:
     @property
     def test_id(self) -> str:
         return self.halter_serial_number
+
+    def _asdict(self) -> dict:
+        return {
```

Comment:
Did you want to be explicit here, or just?
```suggestion
        return dataclasses.asdict(self)
```